examples/tutorials/advanced/websockets-example-MNIST/start_websocket_servers.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+import sys
+from pathlib import Path
+
+python = Path(sys.executable).name
+
+FILE_PATH = Path(__file__).resolve().parents[5].joinpath("run_websocket_server.py")
```

Comment:
I think here should be one level lower (4 instead of 5)

Reply:
corrected.